src/modules/simulation/gz_bridge/module.yaml

```diff
@@ -20,10 +20,10 @@ actuator_output:
       group_label: 'Servos'
       channel_label: 'Servo'
       standard_params:
-        disarmed: { min: 0, max: 1000, default: 500 }
-        min: { min: 0, max: 1000, default: 0 }
-        max: { min: 0, max: 1000, default: 1000 }
-        failsafe: { min: 0, max: 1000 }
+        disarmed: { min: 0, max: 3600, default: 1800 }
+        min: { min: 0, max: 3600, default: 1350 }
+        max: { min: 0, max: 3600, default: 2250 }
+        failsafe: { min: 0, max: 3600 }
```

Comment:
Where do these numbers come from? Can we normalize it for simulation?

Reply:
The goal is to map the full rotation of -180° to 180° (probably the most you could ever want) as a uint16 range of 0 to 3600 because we (still) don't have float mixing which was drafted in https://github.com/PX4/PX4-Autopilot/pull/22488

Reply:
Ah, so this was what I was wondering. I guess it is fine for now, but would wish if we can remove the arbitrary scaling which a lot of people having stumbling over (including myself repeatedly many times).